src/InteractiveBridge.ts

```diff
@@ -29,6 +29,56 @@ export class InteractiveBridge extends LiveryBridge {
     });
   }
 
+  /**
+   * Register `handler` function to be called with `arg` and `listener` when `sendCustomCommand()` is called on
+   * other side with matching `name`.
+   *
+   * @deprecated Will become a protected member in LiveryBridge.ts in the next major version. Use registerPlayerCommand() instead.
+   */
+  public registerCustomCommand(
+    name: string,
+    handler: (arg: unknown, listener: (value: unknown) => void) => unknown,
+  ) {
+    return super.registerCustomCommand(name, handler);
+  }
+
+  /**
+   * Register `handler` function to be called with `arg` and `listener` when sendInteractiveCommand() is called on
```

Comment:
I'd word this: "Register `handler` function to be called with `arg` and `listener` when sendInteractiveCommand() is called from the livery-player side with matching `name`."